packages/twenty-front/src/modules/activities/ask-ai/right-drawer/components/RightDrawerAIChat.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from '@emotion/styled';
+import { useRecoilState } from 'recoil';
+
+import { SQLQueryBuilder } from '@/activities/ask-ai/right-drawer/components/SQLQueryBuilder';
+import { SQLQueryResultTable } from '@/activities/ask-ai/right-drawer/components/SQLQueryResultTable';
+import { askAIQueryState } from '@/activities/ask-ai/right-drawer/states/askAIQueryState';
+import {
+  AutosizeTextInput,
+  AutosizeTextInputVariant,
+} from '@/ui/input/components/AutosizeTextInput';
+import { useGetAskAiQuery } from '~/generated/graphql';
+
+const StyledContainer = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: flex-start;
+  overflow-y: auto;
+  position: relative;
+`;
+
+const StyledChatArea = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  padding: 24px 24px 0px;
+`;
+
+const StyledAskAIQuery = styled.div`
+  font-weight: bold;
+  padding-bottom: 12px;
+`;
+
+const StyledSQLQueryResult = styled.div`
+  overflow-x: scroll;
+  padding-bottom: 24px;
+`;
+
+const StyledNewMessageArea = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0px 24px 24px;
+`;
+
+export const RightDrawerAIChat = () => {
+  const [askAIQuery, setAskAIQuery] = useRecoilState(askAIQueryState);
+
+  const { data, loading } = useGetAskAiQuery({
+    variables: {
+      text: askAIQuery,
+    },
+    skip: !askAIQuery,
+  });
+
+  return (
+    <StyledContainer>
+      <StyledChatArea>
+        {askAIQuery && (
+          <div>
+            <StyledAskAIQuery>{askAIQuery}</StyledAskAIQuery>
+            <SQLQueryBuilder
+              loading={loading}
+              sqlQuery={data?.getAskAI.sqlQuery}
+            />
+            {!loading && (
+              <StyledSQLQueryResult>
+                {typeof data?.getAskAI.sqlQueryResult === 'string' ? (
+                  <SQLQueryResultTable
+                    sqlQueryResult={data.getAskAI.sqlQueryResult}
+                    recordMetadataById={data.getAskAI.recordMetadataById}
+                  />
+                ) : (
+                  'Invalid SQL query.'
```

Comment:
Probably when query is invalid we should let the user have an option to view the error.
E.g. I tried this and it was frustrating not to be able to see the error as it wasn't obvious to me (v2 to add to roadmap: let AI try to fix the error)
<img width="480" alt="Screenshot 2024-06-23 at 10 59 44" src="https://github.com/twentyhq/twenty/assets/6399865/8d56e005-0d43-4861-ac8a-a7d3e89eb380">